code/modules/mob/living/simple_animal/brainbot/brainbotProcs.dm

```diff
@@ -0,0 +1,273 @@
+	set name = "HELP()"
+	set desc = "INJECTS A COCKTAIL OF BENEFICIAL CHEMICALS INTO YOUR HOST. CONTINUED USE MAY CAUSE DAMAGE."
+	set category = "SYSTEM FUNCTIONS"
+	var/cooldown = 0
+
+	var/mob/living/simple_animal/brainbot/user = usr
+	var/mob/living/carbon/human/host = user.controlledMob
+	if(!istype(user) || !user)
+		return
+	if(user.stat)
+		return
+	if(!host)
+		user << "<span class='warning'>YOU HAVE NO HOST.</span>"
+		return
+	if(!host.reagents)
+		user << "<span class='warning'>YOUR HOST IS INCAPABLE OF METABOLIZING REAGENTS.</span>"
+		return
+	if(host.stat == DEAD)
+		user << "<span class='warning'>YOUR HOST IS DEAD AND CANNOT METABOLIZE REAGENTS.</span>"
+		return
+	if(cooldown)
+		user << "<span class='warning'>YOU ARE STILL SYNTHESIZING ANOTHER SERUM.</span>"
+		return
+	user << "<span class='notice'>YOU INJECT A SERUM OF STIMULANTS AND BENEFICIAL NARCOTICS INTO YOUR HOST. IT WILL TAKE SOME TIME TO CREATE MORE.</span>"
+	host << "<span class='info'>You suddenly feel amazing! Your head is crystal-clear!</span>"
+	host.reagents.add_reagent("methamphetamine", 3) //gotta go fast; doesn't inject enough to addict or overdose you
+	host.reagents.add_reagent("ephedrine", 5)
+	host.reagents.add_reagent("stimulants", 5)
+	host.reagents.add_reagent("omnizine", 5)
+	host.reagents.add_reagent("epinephrine", 5)
+	host.reagents.add_reagent("space_drugs", 5)
+	host.reagents.add_reagent("coffee", 5)
+	host.AdjustStunned(-3)
+	host.AdjustWeakened(-3)
+	host.AdjustParalysis(-3)
+	cooldown = 1
+	spawn(1200) //2 minute cooldown
+		cooldown = 0
+		user << "<span class='notice'>YOU HAVE METABOLIZED ANOTHER BENEFICIAL SERUM. YOU MAY NOW USE HELP().</span>"
+
+
+/mob/living/simple_animal/brainbot/proc/harmHost()
+	set name = "BURN()"
+	set desc = "INJECTS A COCKTAIL OF HARMFUL CHEMICALS INTO YOUR HOST. USE TO INVOKE OBEDIENCE."
+	set category = "SYSTEM FUNCTIONS"
+	var/cooldown = 0
+
+	var/mob/living/simple_animal/brainbot/user = usr
+	var/mob/living/carbon/human/host = user.controlledMob
+	switch(alert("ARE YOU SURE? THIS WILL LIKELY KILL YOUR HOST.",,"CONFIRM","CANCEL"))
+		if("CANCEL")
+			return 0
+	if(!istype(user) || !user)
+		return
+	if(user.stat)
+		return
+	if(!host)
+		user << "<span class='warning'>YOU HAVE NO HOST.</span>"
+		return
+	if(!host.reagents)
+		user << "<span class='warning'>YOUR HOST IS INCAPABLE OF METABOLIZING REAGENTS.</span>"
+		return
+	if(host.stat == DEAD)
+		user << "<span class='warning'>YOUR HOST IS DEAD AND CANNOT METABOLIZE REAGENTS.</span>"
+		return
+	if(cooldown)
+		user << "<span class='warning'>YOU ARE STILL SYNTHESIZING ANOTHER SERUM.</span>"
+		return
+	user << "<span class='notice'>YOU INJECT A SERUM OF TOXINS INTO YOUR HOST. IT WILL TAKE SOME TIME TO CREATE MORE.</span>"
+	host << "<span class='boldannounce'>Your head throbs sharply, and you suddenly are engulfed in flames!</span>"
+	host.reagents.add_reagent("clf3", 5)
+	host.reagents.add_reagent("phlogiston", 5)
```

Comment:
CLF3 has a super high metabolism rate, I'd reccomend bumping these all up to 15 units.